D4RL (Mujoco).md

```diff
@@ -0,0 +1,343 @@
+D4RL (Mujoco)
+~~~~~~~~~~~~~
+
+ABSTRACT
+=======
+D4RL is an open source benchmark for offline Reinforcement Learning, which provides a standardized environment and dataset for training and benchmarking algorithms. The collecting strategy of datasets includes:
+
+1. being generated by hand-designed rules and expert demonstrations
+2. agents performing different tasks in the same environment
+3. using hybrid strategies
+
+Specifically, the following seven sub-environments are included
+
+-  Maze2D
+-  AntMaze
+-  Gym-MuJoco
+-  Adroit
+-  FrankaKitchen
+-  Flow
+-  Offline CARLA
+
+Note: offline RL is the dataset with d4rl for training, and tests are interacted with a specific RL environment, such as Mujoco.
```

Comment:
"offline RL is trained on the d4rl dataset, and tested by interacting with a specific ..."